x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/util/StringUtils.java

```diff
@@ -29,11 +26,4 @@ public static LikePattern toLikePattern(String s) {
 
         return new LikePattern(likeString, escape);
     }
-
-    public static LikePattern toLikePattern(Expression expression) {
```

Comment:
not strictly required by change, but had to be inspected and it's not used anywhere.

Reply:
++, good find!